packages/checkbox-group/define.js

```diff
@@ -0,0 +1,7 @@
+import { LionCheckboxGroup } from './src/LionCheckboxGroup.js';
+import { LionCheckboxIndeterminate } from './src/LionCheckboxIndeterminate.js';
+import { LionCheckbox } from './src/LionCheckbox.js';
+
+customElements.define('lion-checkbox-group', LionCheckboxGroup);
+customElements.define('lion-checkbox-indeterminate', LionCheckboxIndeterminate);
+customElements.define('lion-checkbox', LionCheckbox);
```

Comment:
```suggestion
import '@lion/checkbox-group/define-checkbox-group';
import '@lion/checkbox-group/define-checkbox';
import '@lion/checkbox-group/define-checkbox-indeterminate';
```

so then end user can do

```js
import '@lion/checkbox-group/define';

// or

import '@lion/checkbox-group/define-checkbox-group';
```